source/opt/eliminate_dead_functions_util.cpp

```diff
@@ -24,6 +24,7 @@ Module::iterator EliminateFunction(IRContext* context,
   (*func_iter)
       ->ForEachInst([context](Instruction* inst) { context->KillInst(inst); },
                     true);
+  // TODO: Kill DebugInlinedAt used only by DebugScope in a killed function.
```

Comment:
This can be taken care of by ADCE.  Nothing needs to be done in this pass for this.